src/test/java/com/squareup/kotlinpoet/FunSpecTest.kt

```diff
@@ -56,14 +54,9 @@ class FunSpecTest {
     return elements.getTypeElement(`class`.canonicalName)
   }
 
-  private fun findFirst(elements: Collection<ExecutableElement>, name: String): ExecutableElement {
-    for (executableElement in elements) {
-      if (executableElement.simpleName.toString() == name) {
-        return executableElement
-      }
-    }
-    throw IllegalArgumentException(name + " not found in " + elements)
-  }
+  private fun findFirst(elements: Collection<ExecutableElement>, name: String) =
+      elements.firstOrNull { it.simpleName.toString() == name } ?:
+          throw IllegalArgumentException(name + " not found in " + elements)
```

Comment:
could also be `requireNotNull(elements.firstOrNull { it.simpleName.toString() == name }) { name + " not found in " + elements }`